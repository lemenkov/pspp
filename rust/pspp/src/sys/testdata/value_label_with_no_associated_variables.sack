# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; 1; 1; 0; -1; 100.0; "01 Jan 11"; "20:53:52"; s64 ""; i8 0 *3;

# Numeric variable.
2; 0; 0; 0; 0x050800 *2; s8 "NUM1";

# Value label with no variables.
3; 1; 1.0; i8 3; s7 "one"; 4; >>0<<;

# End of dictionary.
>>999; 0<<;
