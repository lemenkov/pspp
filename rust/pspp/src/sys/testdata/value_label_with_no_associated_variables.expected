At offset 0xe8, at least one valid variable index for value labels is required but none were specified.

This system file does not indicate its own character encoding.  For best results, specify an encoding explicitly.  Use SYSFILE INFO with ENCODING="DETECT" to analyze the possible encodings.

╭──────────────────────┬────────────────────────╮
│       Created        │    01-JAN-2011 20:53:52│
├──────────────────────┼────────────────────────┤
│Writer Product        │PSPP synthetic test file│
├──────────────────────┼────────────────────────┤
│       Compression    │SAV                     │
│       Number of Cases│Unknown                 │
╰──────────────────────┴────────────────────────╯

╭─────────┬─╮
│Variables│1│
╰─────────┴─╯

╭────┬────────┬─────┬─────────────────┬─────┬─────┬─────────┬────────────┬────────────┬──────────────╮
│    │Position│Label│Measurement Level│ Role│Width│Alignment│Print Format│Write Format│Missing Values│
├────┼────────┼─────┼─────────────────┼─────┼─────┼─────────┼────────────┼────────────┼──────────────┤
│num1│       1│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
╰────┴────────┴─────┴─────────────────┴─────┴─────┴─────────┴────────────┴────────────┴──────────────╯
